src/api/fetchRecipes.js

```diff
@@ -5,7 +5,7 @@ const apiKey = process.env.REACT_APP_RECIPES_APP_KEY;
 
 const recipesUrl = "/recipes/v2";
 
-const fetchRecipes = async (ingredients, calories) => {
+const fetchRecipes = async (ingredients, calories = 5000) => {
```

Comment:
Shouldn't `ingredients` be initialised as an empty string? Pretty sure it's `undefined` otherwise